.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: java
 
 jdk:
-  - oraclejdk7
-  - openjdk6
```

Comment:
This is removing support for Java 7 and JDK 6. Why? Is that what's breaking the builds? Did Travis removed support for them?

Reply:
I guess we can add back Java 7, but Java 6 has reached EOL 7 years ago: https://docs.travis-ci.com/user/reference/precise/#jvm-clojure-groovy-java-scala-vm-images
However I would think hard about supporting Java 6, I wouldn't do that - it's better to get users to move on.

Reply:
Is that breaking the build at all? If not, then can you please create a new PR running the tests only in the newer versions of JDK and also update [the README](https://github.com/js-cookie/java-cookie/tree/e16402cc114c558c0f3a10acb68f14169e7cde99#java-cookie-) where it says "Supports Java 6+, Servlet 2.2+"?

Reply:
I'm happy to upgrade

Reply:
I added all available JDKs on Travis, including JDK6. Let's see how it does.

Reply:
Unfortunately JDK6 and JDK7 fail on Travis, JDK7 do to unsupported encryption when installing NPM via frontend-maven-plugin. I removed both and updated the README to require at least JDK8. As the current version of `java-cookie` is only `0.0.2`, we can bump the version e.g. to `1.0.0` to signal this breaking change (after adding the other planned fixes). 😉